Code/Tools/ProjectManager/Source/GemCatalog/GemInspector.cpp

```diff
@@ -30,6 +30,7 @@ namespace O3DE::ProjectManager
         setVerticalScrollBarPolicy(Qt::ScrollBarAsNeeded);
 
         m_mainWidget = new QWidget();
+        m_mainWidget->setFixedWidth(240);
```

Comment:
`sidePanelWidth` is defined and used in GemCatalogScreen.cpp also 240. Maybe we should define this in one place and use it everywhere?